src/main/kotlin/no/nb/bikube/newspaper/controller/TitleController.kt

```diff
@@ -39,10 +46,27 @@ class TitleController (
     ])
     fun createTitle(
         @RequestBody title: Title
-    ): ResponseEntity<Flux<Title>> {
-        if (title.name.isNullOrEmpty()) {
-            return ResponseEntity.badRequest().build()
+    ): Mono<ResponseEntity<Title>> {
+        return if (title.name.isNullOrEmpty()) {
+            Mono.error(BadRequestBodyException("Title name cannot be null or empty"))
+        } else if (title.startDate != null && title.endDate != null && title.startDate.isAfter(title.endDate)) {
+            Mono.error(BadRequestBodyException("Start date cannot be after end date"))
+        } else {
+            val publisherMono: Mono<Publisher> = title.publisher?.let {
+                axiellService.createPublisher(it).onErrorResume { Mono.empty() }
```

Comment:
Should separate between different failures. What if we just can't create at this moment?